### PR TITLE
Account management kyc kyb

### PR DESCRIPTION
**Pull request by shan-57blocks**

### CODE REVIEW

File: packages/huma-shared/src/services/IdentityServiceV2.ts [hunk 1]
```diff
@@ -239,52 +239,12 @@ const approveLender = async (
...
-  chainId: number,
-  isDev = false,
-): Promise<HumaAccount> =>
-  requestGet<HumaAccount>(
-    `${configUtil.getIdentityAPIUrl(
-      chainId,
-      isDev,
-    )}/wallets/${walletAddress}/account?chainId=${chainId}`,
+    )}/account/wallets/${chainId}/${walletAddress}`,
+    { trancheAddress: contractAddress, chain_specific_data: chainSpecificData },
```
Comment (ljiatu) [inline packages/huma-shared/src/services/IdentityServiceV2.ts:247]: nit: The `chain_specific_data` key could be in camelCase.

File: packages/huma-shared/src/services/IdentityServiceV2.ts [hunk 2]
```diff
@@ -87,6 +86,16 @@ export type HumaAccount = {
   }
 }
 
+/**
+ * Object representing the Huma account login result.
+ * @typedef {Object} HumaAccountLoginResult
+ * @property {HumaAccount} account The Huma account.
+ * @property {boolean} isNewAccount Is new account or not.
+ */
+export type HumaAccountLoginResult = HumaAccount & {
```
Comment (ljiatu) [inline packages/huma-shared/src/services/IdentityServiceV2.ts:95]: FYI I'm adding a new field called `account` into the `LoginResult` in this PR: https://github.com/00labs/huma-py/pull/238

**Review by ljiatu — APPROVED** [review]
LGTM % comments

File: packages/huma-shared/src/services/IdentityServiceV2.ts (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/services/IdentityServiceV2.ts]: Updated

File: packages/huma-shared/src/services/IdentityServiceV2.ts (hunk 2 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/services/IdentityServiceV2.ts:95]: Will update in a later PR

**Pull request merged**